book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rent_book(person, date)
+    Rental.new(date, person, self)
```

Comment:
- [ ] We already have access to the `book` object through `self`, so what we need is `person`, not `book`.

- [ ] Kindly take note of the arrangement for the parameters.

  ```
  def rent_book(person, date)
    Rental.new(date, self, person)
  ```

Reply:
Thank you for the tip.